extensions/machine-learning-services/src/dialogs/externalLanguages/languageContentView.ts

```diff
@@ -0,0 +1,159 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as mssql from '../../../../mssql/src/mssql';
+import { LanguagesDialogModel } from './languagesDialogModel';
+import { LanguageViewBase } from './languageViewBase';
+import * as constants from '../../common/constants';
+
+export class LanguageContentView extends LanguageViewBase {
+
+	private _serverPath: azdata.RadioButtonComponent;
+	private _localPath: azdata.RadioButtonComponent;
+	private _extensionFile: azdata.TextComponent;
+	private _extensionFileName: azdata.TextComponent;
+	private _envVariables: azdata.TextComponent;
+	private _parameters: azdata.TextComponent;
+	private _isLocalPath: boolean = true;
+
+	/**
+	 *
+	 */
+	constructor(
+		parent: LanguageViewBase,
+		model: LanguagesDialogModel,
+		private _modelBuilder: azdata.ModelBuilder,
+		private _formBuilder: azdata.FormBuilder,
+		private _languageContent: mssql.ExternalLanguageContent | undefined,
+	) {
+		super(model, parent);
+		this._localPath = this._modelBuilder.radioButton()
+			.withProperties({
+				value: 'local',
```

Comment:
>local [](start = 12, length = 5)

These should be localized

Reply:
this is the value of the dropdown not the label

---
In reply to: [375360037](https://github.com/microsoft/azuredatastudio/pull/9043#discussion_r375360037) [](ancestors = 375360037)